apps/examples/src/examples/pin-bindings/PinExample.tsx

```diff
@@ -93,7 +97,9 @@ class PinShapeUtil extends ShapeUtil<PinShape> {
 			.getShapesAtPoint(pageAnchor, { hitInside: true })
 			.filter(
 				(shape) =>
-					shape.type !== 'pin' && shape.parentId === pin.parentId && shape.index < pin.index
+					this.editor.canBindShapes(pin.id, shape.id, 'pin') &&
+					shape.parentId === pin.parentId &&
+					shape.index < pin.index
```

Comment:
Did we lose the constraint that pins can't bind to other pins?

Reply:
nope, it's just covered by the `canBindShapes` check now